api/HTMLElement.json

```diff
@@ -599,10 +599,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": "6"
+              "version_added": "3"
```

Comment:
Since Safari 3 was the earliest version tested for this PR, can we really say this? Let's use &le;4 where it's not clear what the status of Safari 1-2 were.

Reply:
The `HTMLElement` API wasn't supported until Safari 3, so "1-2" is not possible -- I wouldn't set it so confidently otherwise.